OMCompiler/Compiler/Template/CodegenCpp.tpl

```diff
@@ -13187,7 +13170,29 @@ match simVar
     if createDebugCode then
        'double& <%jacobianVar%>;' else
        '#define <%jacobianVar%> _<%matrixName%><%typeName%>'
-  end match
+  */
+  case SIMVAR(numArrayElement = {}, arrayCref = NONE()) then
+    if createDebugCode then
+      'not yet implemented'
```

Comment:
Since we can't throw an error from Susan you can add a unique error message that states where this "error" comes from.
Something like `template jacobianVarDefine: case SIMVAR(numArrayElement = {}, arrayCref = NONE()) not implemented`

Reply:
Ok, I'll put some thought into creating debug code, also regarding the case above that's commented out. On second look it shouldn't be too hard to figure out.